roles/kubernetes/preinstall/tasks/resolvconf.yml

```diff
@@ -4,17 +4,33 @@
   register: resolvconf
   ignore_errors: yes
   changed_when: false
+  tags: facts
+
+- name: check kubelet
```

Comment:
The better check is if there's a kubelet service running, but this is almost as good.